modular_skyrat/modules/customization/modules/mob/living/carbon/human/species/akula.dm

```diff
@@ -81,3 +81,14 @@
 
 /datum/species/akula/get_species_lore()
 	return list(placeholder_lore)
+
+/datum/species/akula/prepare_human_for_preview(mob/living/carbon/human/human)
+	var/main_color = "#394b66"
+	var/secondary_color = "#818b9b"
+	human.dna.features["mcolor"] = main_color
+	human.dna.features["mcolor2"] = secondary_color
+	human.dna.features["mcolor3"] = secondary_color
+	human.dna.species.mutant_bodyparts["tail"] = list(MUTANT_INDEX_NAME = "Shark", MUTANT_INDEX_COLOR_LIST = list(main_color, secondary_color, secondary_color))
+	human.dna.species.mutant_bodyparts["snout"] = list(MUTANT_INDEX_NAME = "hShark", MUTANT_INDEX_COLOR_LIST = list(main_color, secondary_color, secondary_color))
```

Comment:
Don't Akulas also have like, a fin or something, on their head? That portrait looks really odd at the moment, any way to make it look less like an egg would be good in my books.

Reply:
looked into it, found some pretty cursed knowledge: they DONT. They have ears, but-

`/datum/sprite_accessory/ears/mutant/akula/sergal`

They're SERGAL ears. `/mutant/akula` is just an empty basetype that defines the recommended species, `/mutant/akula/sergal` ears are the only ones with a texture that are technically akula ears

Do with that what you will